changelog/README.rst

```diff
@@ -8,7 +8,7 @@ Changelog
 
 This directory contains "news fragments" which are short files that contain a small **ReST**-formatted text that will be added to the next ``CHANGELOG``.
 
-The ``CHANGELOG`` will be read by users, so this description should be aimed at sunraster users instead of describing internal changes which are only relevant to the developers.
+The ``CHANGELOG`` will be read by users, so this description should be aimed at SunPy users instead of describing internal changes which are only relevant to the developers.
```

Comment:
```suggestion
The ``CHANGELOG`` will be read by users, so this description should be aimed at sunraster users instead of describing internal changes which are only relevant to the developers.
```